import_tracker/import_tracker.py

```diff
@@ -377,7 +413,7 @@ def _figure_out_import(
     log.debug3("Module file: %s", getattr(mod, "__file__", None))
     if not import_name:
         import_name = root_mod_name
-    else:
+    elif root_mod_name:
```

Comment:
This change showed up as I was trying to understand why some modules were not identified as optional. I _believe_ this was a bug since the resulting name could be `".foobar"` which would not show up in `sys.modules` when `"foobar"` would.